org.kde.krita.yaml

```diff
@@ -0,0 +1,681 @@
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+      - -DPNG_TESTS=OFF
+    sources:
+      - type: archive
+        url: https://download.sourceforge.net/libpng/libpng-1.6.37.tar.gz
+        sha256: daeb2620d829575513e35fecc83f0d3791a620b9b93d800b763542ece9390fb4
+  - name: tiff
+    buildsystem: cmake-ninja
+    builddir: true
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+    sources:
+      - type: archive
+        url: https://download.osgeo.org/libtiff/tiff-4.2.0.tar.gz
+        sha256: eb0484e568ead8fa23b513e9b0041df7e327f4ee2d22db5a533929dfc19633cb
+      - type: patch
+        paths:
+          - patches/tiff/CVE-2018-12900.patch
+          - patches/tiff/0001-Enable-writing-Photoshop-blobs.patch
+  - name: jpeg
+    buildsystem: cmake-ninja
+    builddir: true
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+      - -DWITH_SIMD=OFF
+    sources:
+      - type: archive
+        url: https://downloads.sourceforge.net/project/libjpeg-turbo/2.0.6/libjpeg-turbo-2.0.6.tar.gz
+        sha256: d74b92ac33b0e3657123ddcf6728788c90dc84dcb6a52013d758af3c4af481bb
+  - name: boost
+    buildsystem: simple
+    build-commands:
+      - ./bootstrap.sh --prefix=/app --with-libraries=system
+      - ./b2 install -j $FLATPAK_BUILDER_N_JOBS
+    sources:
+      - type: archive
+        url: https://boostorg.jfrog.io/artifactory/main/release/1.71.0/source/boost_1_71_0.tar.bz2
+        sha256: d73a8da01e8bf8c7eda40b4c84915071a8c8a0df4a6734537ddde4a8580524ee
+  - name: fftw3
+    buildsystem: cmake-ninja
+    builddir: true
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+      - -DBUILD_TESTS=OFF
+      - -DENABLE_THREADS=ON
+    sources:
+      - type: archive
+        url: http://fftw.org/fftw-3.3.9.tar.gz
+        sha256: bf2c7ce40b04ae811af714deb512510cc2c17b9ab9d6ddcf49fe4487eea7af3d
+  - name: eigen3
+    buildsystem: cmake-ninja
+    builddir: true
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+      - -DBUILD_TESTING=OFF
+    cleanup:
+      - '*'
+    sources:
+      - type: archive
+        url: https://gitlab.com/libeigen/eigen/-/archive/3.3.9/eigen-3.3.9.tar.gz
+        sha256: 7985975b787340124786f092b3a07d594b2e9cd53bbfe5f3d9b1daee7d55f56f
+  - name: expat
+    buildsystem: cmake-ninja
+    builddir: true
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+    sources:
+      - type: archive
+        url: https://github.com/libexpat/libexpat/releases/download/R_2_2_9/expat-2.2.9.tar.gz
+        sha256: 4456e0aa72ecc7e1d4b3368cd545a5eec7f9de5133a8dc37fdb1efa6174c4947
+  - name: exiv2
+    buildsystem: cmake-ninja
+    builddir: true
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+      - -DEXIV2_BUILD_SAMPLES=OFF
+      - -DEXIV2_ENABLE_NLS=OFF
+      - -DEXIV2_BUILD_EXIV2_COMMAND=OFF
+    cleanup:
+      - /bin
+    sources:
+      - type: archive
+        url: https://mirrors.ocf.berkeley.edu/kde-applicationdata/krita/build/dependencies/exiv2-0.27.3-Source.tar.gz
```

Comment:
0.27.5 has been out for a bit. and it allow dropping the patch below AFAIK (breakage on aarch64)